automated-comments/go/raindrops/entry_signature_changed.md

```diff
@@ -0,0 +1,4 @@
+The signature of the `Convert` function should be `Convert(i int) string`.
+Exercism is built on the TDD (Test Driven Development) principle. 
```

Comment:
Should we perhaps link to a page that has a basic introduction of TDD?

Reply:
What do you think about this one?
http://testfirst.org/

It's mostly about Test Driven Teaching. Basically what exercism.io does.

TDD is mostly about writing the tests first which the students don't do here.

Reply:
Maybe this direct link into the website: http://testfirst.org/about#tdd

Reply:
Or just Wikipedia? https://en.wikipedia.org/wiki/Test-driven_development

Reply:
Either could work, although I have a slight preference for the http://testfirst.org/about#tdd one, due to its brevity.